index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+    <title>Apple</title>
+    <link rel="stylesheet" href="assets/styles.css">
+    <script src="https://kit.fontawesome.com/4f5233a2df.js" crossorigin="anonymous"></script>
+    <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <div class="bodycont">
+        <div class="mainimg">
+            <nav class="navbar">
+                <form class="menu">
+                    <button class="menuitem2 logo"> </button>
+                    <button class="menuitem2 store"></button>
+                    <button class="menuitem2 mac"></button>
+                    <button class="menuitem2 ipod"></button>
+                    <button class="menuitem2 iphone"></button>
+                    <button class="menuitem2 ipad"></button>
+                    <button class="menuitem2 itunes"></button>
+                    <button class="menuitem2 support"></button>
+                    <div class="menuitem2 searchbar"> <input type="text" class="bar"></div>
+                </form>
+            </nav>
+            <main class="container">
+                <div class="maintxt">
+                    <h1 class="maintitle">What will your verse be?</h1>
+                    <h2> Everyone has something to share. See how people are using iPad in their unique ways to add to
+                        the
+                        world's story.</h2>
+
+                    <a class="linkyv"
+                        href="https://web.archive.org/web/20140301004610/http:/www.apple.com/your-verse/#video-your-verse">
+                        <div class="yv"></div>
+                    </a>
+                </div>
+            </main>
+        </div>
+
+        <aside class="relimg">
+            <div class="relcontainer">
+                <img alt="iPhone 5S" src="assets/promo_iphone5s.jpg">
+                <img alt="iPhone 5c" src="assets/promo_iphone5c.jpg">
+                <img alt="30 years of Mac" src="assets/promo_30_years.jpg">
+                <img alt="supplier responsibility" src="assets/promo_supplier_responsibility.jpg">
```

Comment:
These four elements should be wrapped in anchor tags `<a>`

Reply:
Done, will resubmit for review